### PR TITLE
createPersona()

### PR DESCRIPTION
**Pull request by Burdishka**
[Trello card link](https://trello.com/c/RB9GsmhQ/990-calling-createpersona-method)

calling out ProjectUtils.Portal.createPersona() in admin.permissions.forUser test classes

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

File: src/test/java/test/admin/permissions/forUser/AdminPermissionsForUsersMenuTest.java [hunk 1]
```diff
@@ -31,14 +32,11 @@
...
-
         urlAndPasswordArray = createInstanceAndLoginWithPassw(getDriver());
+
         ConfigurationPage configurationPage = new ConfigurationPage(getDriver());
 
-        Assert.assertTrue(configurationPage
-                .clickConfiguration()
-                .clickNewPersona()
-                .createPersona(PERSONA_NAME)
+        Assert.assertTrue(ProjectUtils.Portal.createPersona(getDriver(), PERSONA_NAME)
```
Comment (oooleks) [inline src/test/java/test/admin/permissions/forUser/AdminPermissionsForUsersMenuTest.java:39]: No assert needed for the test which is not testing user creation.

File: src/test/java/test/admin/permissions/forUser/AdminPermissionsForUserScreensTest.java [hunk 2]
```diff
@@ -18,15 +18,10 @@
     public void createPermissionSittingForUsersScreensTest() {
         ProjectUtils.Portal.createInstanceAndLogin(getDriver());
 
-        ConfigurationPage configurationPage = new ConfigurationPage(getDriver());
-
-        Assert.assertTrue(configurationPage
-                .clickConfiguration()
-                .clickNewPersona()
-                .createPersona(PERSONA_NAME)
+        Assert.assertTrue(ProjectUtils.Portal.createPersona(getDriver(), PERSONA_NAME)
```
Comment (oooleks) [inline src/test/java/test/admin/permissions/forUser/AdminPermissionsForUserScreensTest.java:21]: No assert for user creation needed for this test

**Review by guram21 — APPROVED** [review]

File: src/test/java/test/admin/permissions/forUser/AdminPermissionsForUsersMenuTest.java (hunk 1 above)
Comment (Burdishka) [reply src/test/java/test/admin/permissions/forUser/AdminPermissionsForUsersMenuTest.java]: done

File: src/test/java/test/admin/permissions/forUser/AdminPermissionsForUserScreensTest.java (hunk 2 above)
Comment (Burdishka) [reply src/test/java/test/admin/permissions/forUser/AdminPermissionsForUserScreensTest.java]: done

**Review by oooleks — APPROVED** [review]

**Pull request merged**